### PR TITLE
Add more badges to README

### PR DESCRIPTION
**Pull request by bittner**
This PR converts the README from MarkDown to Python's preferred reStructuredText, and adds a few nice buttons on top of the README.

To make all buttons work correctly some accounts need to be created or repos activated in the respective services:

* [Landscape](https://landscape.io/github/jrief/djangocms-cascade/master) (static code analysis on Python code)
* [Gitter](https://gitter.im/jrief/djangocms-cascade) (a chat room attached to your Git repository)

### CODE REVIEW

**Comment by rfleschenberg** [comment]
Hello @bittner. Thank you for the PR!

I am not sure if we want to switch to RST for the README. Many people consider Markdown more nice in some regards, and we actually have some code in `setup.py` to convert the README to RST on the fly. This is also the reason why your PR breaks the CI build:

```
      File "/home/travis/build/jrief/djangocms-cascade/setup.py", line 13, in convert
        with io.open(filename, encoding='utf-8') as fd:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
```

The spelling corrections look good to me. About the badges, I don't really have an opinion. Maybe it would be better if you create separate PRs for the changes. Before we can merge this, we would have to fix the build failure. But I'd like to wait for @jrief's opinion here.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-## NEWS for version 0.9
+**DjangoCMS-Cascade** is the Swiss army knife for working with Django CMS plugins.
 
-* Compatible with django-CMS 3.3.0
-* Important notice: Converted ``SharableCascadeElement`` into a proxy model. A migration
-  merges these two models, so please backup your database before upgrading.
-* Add support for Section Bookmarks. This allows to add links on subsections of a CMS page.
-* Fixes many minor issues, check the changelog for details.
-* Replaced configuration settings ``CMSPLUGIN_CASCADE_LINKPLUGIN_CLASSES`` against
-  ``CMSPLUGIN_CASCADE['link_plugin_classes']`` for better consistency.
```
Comment (rfleschenberg) [inline README.md]: I think we should not delete this information. Can you please re-add these lines? Then we can probably merge this. Thanks!

Comment (bittner) [inline README.md]: Isn't the _"Please see &lt;link to release notes&gt; before upgrading from an older version"_ more explicit? I felt this is what the excerpt from the change log is meant to point out.

The paragraph is also duplicated in the docs, even worse, it has diverged in the meantime (it looks like). I have tried to add the information that went missing to the change log document. Does that make sense?

Comment (rfleschenberg) [inline README.md]: Ahh, I missed that, sorry. I'll merge this, looks good to me.

**Pull request merged**